repo_guidelines.md

```diff
@@ -132,6 +136,60 @@ See the [`LICENSE`](/LICENSE) file for details.
  */
 ```
 
+### Core Issue Label Configuration
+EDGI repos use a minimum/core set of standard issue labels.
+
+Labels are applied using the NPM package [github-labels](https://www.npmjs.com/package/github-labels) using the following `configuration.json`:
+
+```json
+[
+  {
+    "name": "blocked",
+    "color": "F73E34"
+  },
+  {
+    "name": "coordination",
+    "color": "2B70C4"
+  },
+  {
+    "name": "documentation",
+    "color": "6FEB73"
+  },
+  {
+    "name": "good-first-issue",
+    "color": "1A610B"
+  },
+  {
+    "name": "idea",
+    "color": "71C8FA"
+  },
+  {
+    "name": "infrastructure",
+    "color": "F7F01B"
+  },
+  {
+    "name": "[priority-★★★]",
+    "color": "ff7f00"
+  },
+  {
+    "name": "[priority-★★☆]",
+    "color": "ffaa00"
+  },
+  {
+    "name": "[priority-★☆☆]",
+    "color": "ffd400"
+  },
+  {
+    "name": "question",
+    "color": "ED9AA9"
+  },
+  {
+    "name": "stale",
+    "color": "E6E3E3"
+  }
+]
+```
+
 ### Stale Issues Configuration Template
```

Comment:
This is not in the PR, but can we add `- idea` under `exemptLabels` in the  config template?

Reply:
Personally, I'd actually prefer that `idea` issues get picked up by stalebot. Ideas go stale too– something that seemed like a good idea 3 months ago may not now. Or, three months later might be a better moment to take another look at the idea– and the bot might prompt that! Likely that different people feel differently on how this should be handled. Happy to discuss in an issue if you'd like?

Reply:
Oh, we have a misunderstanding and I didn't communicate this well before

In WM, `idea` is used **only** to exempt issues from stalebot. It isn't applied it to anything else. WM was trying to find a succinct term that signified an issue was far off and not have stalebot ping us anymore. In retrospect, reading your comment, a less generic term should've been used, but it's functioned this way since we started using stalebot 😳 

I was suggesting this addition because we were adding the `idea` label in this PR and thought we should add that exemption function to go with the label as well

Hope that clarifies things! Not sure how we should move on this, but, yes, please open an issue to discuss further

Reply:
broken out & resolved in https://github.com/edgi-govdata-archiving/overview/issues/231